crates/bevy_render/src/render_resource/buffer_vec.rs

```diff
@@ -64,6 +64,13 @@ impl<T: NoUninit> RawBufferVec<T> {
         self.buffer.as_ref()
     }
 
+    #[inline]
+    pub fn binding(&self) -> Option<BindingResource> {
```

Comment:
Probably want a doc comment here too

Reply:
I originally didn't add it because the other functions didn't have one, but I see no reason why that is the case so I commented all other functions too.